Makefile

```diff
@@ -68,13 +67,10 @@ modules: FORCE
 
 runtime: FORCE
 	cd runtime && $(MAKE)
-
-llvm-runtime-if-needed: FORCE
 	-@if [ "llvm" = `${CHPL_MAKE_HOME}/util/chplenv/chpl_llvm.py` ]; then \
-	echo "Building runtime for chpl --llvm"; \
+	echo "Building runtime for chpl --llvm" ; \
```

Comment:
Why do we echo this for llvm? Was this some debug that can/should be removed? If not, should it be added to re2 and gmp?